recipes/openorb-eph-builder/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# conda's gfortran on Linux does not install a binary named 'gfortran'
+echo "gfortran = $(which gfortran)"
+[[ ! -f "$BUILD_PREFIX/bin/gfortran" && ! -z "$GFORTRAN" ]] && ln -s "$GFORTRAN" "$BUILD_PREFIX/bin/gfortran"
+echo "gfortran = $(which gfortran)"
```

Comment:
conda-forge is transitioning the way in which it deals with compilers. In the near future, build scripts should always use `$FC` as their FORTRAN compiler. But the older compiler packages don't set this variable. Would a construct like this work with your configure script?

```
if [ -z "$FC" ] ; then
    export FC=gfortran
fi

./configure "$FC" opt --prefix="$PREFIX"
```

?

Reply:
Yes, I'm also unhappy with the present hack; the issue is that the upstream code looks for a compiler by name (i.e., its `configure` script generates a `Makefile.include` that says `FC=gfortran`).

The long-term solution is to modernize oorb's build system, after which this hack won't be necessary. On my TODO list.